docs/integration/javascript.rst

```diff
@@ -112,6 +112,31 @@ When clicking `Test it` again, we can see the JSON structure of the variable dat
 The *contract* and *salesRepresentative* variables have complex types, :ref:`type-file` and :ref:`type-user`, so the table only shows an ID.
 The *Updated value* column shows the result of assigning new values to these variables in the script.
 
+To access :ref:`file <type-file>` content, you need to require the ``files`` API.
+In this example, ``contract`` is a file variable, which has to be activated for the script task.
+
+.. code:: js
+
+  const files = require('files')
+  const fileContent = files.getContent(contract)
+
+The following example loads a CSV file and parses its content:
+
+.. code:: js
+
+  const files = require('files')
+  const csv = require('csv')
+
+  const csvFile = files.getContent(cSV.id)
```

Comment:
Sorry, but I just noticed that now. Could you please change `cSV.id` to something more expressive like `reportCsv` and mention that `reportCsv` is an activated workflow variable?